docs/operating-scylla/admin-tools/scylla-sstable.rst

```diff
@@ -674,6 +674,18 @@ The content is dumped in JSON, using the following schema when ``--vnodes`` comm
 
     $SHARD_ID := Uint
 
+upload
+^^^^^^
+
+Uploads one or more SSTables from local storage to object storage.
+
+.. code-block:: console
+
+    scylla sstable upload \
+      --endpoint s3.us-east-2.amazonaws.com \
+      --object-store-config /etc/scylladb/conf/object-store.yaml \
+      --to bucket-foo/ks/cf /path/to/me-14-big-Data.db
+
```

Comment:
Ah, it's not the same as Cassandra's import.

Suggestions:

1. Add a protocol, so we have s3://bucket/path and file://path/to/files (and https://host/path)
2. Allow providing a path to be scanned rather than having to specify each file

Reply:
Also: need to specify the keyspace/table

Reply:
And it should be part of nodetool!

Reply:
> And it should be part of nodetool!

you mean this change is inconsistent with the title , or this command should be part of nodetool?

Reply:
> Ah, it's not the same as Cassandra's import.
> 
> Suggestions:
> 
> 1. Add a protocol, so we have s3://bucket/path and file://path/to/files (and https://host/path)

it's already supported when specifying `--to`.

> 2. Allow providing a path to be scanned rather than having to specify each file

@denesb what do you think?

Reply:
> Also: need to specify the keyspace/table

@avikivity what would we need keyspace/table for? if we want to be compatible with the scylla manager (SM for short), and expect it to consume the uploaded the sstables, we would need to use a the "path" compatible with it. but we should / cannot compose one by hardwiring **its** naming schema in scylla. because the way how SM composes the keys for sstables is its internals, scylla should not replicate it in its implementation. otherwise it'd be fragile and confusing -- it just hurts the maintainability.